src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -2072,17 +2073,34 @@ public void we_can_collect_the_dropped_messages_stats_from_the_seed_node() throw
             if (Response.Status.OK.getStatusCode() == response.getStatus() && StringUtils.isNotBlank(responseData)) {
               List<DroppedMessages> dropped = SimpleReaperClient.parseDroppedMessagesJSON(responseData);
               if (dropped.isEmpty()) {
-                LOG.error("Got empty response from {}", seed);
+                LOG.error("Got empty DroppedMessages response from {}", seed);
```

Comment:
Issue: do we want the test to fail out at this point? I think maybe we do.

Reply:
I'm failing fast now